### PR TITLE
chore: run prettier

### PR DESCRIPTION
**Pull request by bsmth**

### CODE REVIEW

**Review by tris0laris — APPROVED** [review]

File: yarn.lock [hunk 1]
```diff
@@ -11006,9 +11006,9 @@ url-parse-lax@^3.0.0:
     prepend-http "^2.0.0"
 
 url-parse@^1.4.3, url-parse@^1.5.3:
-  version "1.5.10"
-  resolved "https://registry.yarnpkg.com/url-parse/-/url-parse-1.5.10.tgz#9d3c2f736c1d75dd3bd2be507dcc111f1e2ea9c1"
-  integrity sha512-WypcfiRhfeUP9vvF0j6rw0J3hrWrw6iZv3+22h6iRMJ/8z1Tj6XfLP4DsUix5MhMPnXpiHDoKyoZ/bdCkwBCiQ==
+  version "1.5.7"
```
Comment (bluestreak01) [inline yarn.lock:11009]: is this meant to downgrade ?

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (bsmth) [reply yarn.lock:11009]: yep

Comment (bsmth) [reply yarn.lock:11009]: Netlify builds are not passing due to this error: https://app.netlify.com/sites/unruffled-blackwell-31bfb2/deploys/622b862f037e5c0009c6abc4#L1018

Reverting dependabot's changes seems to fix.

Comment (bluestreak01) [reply yarn.lock:11009]: Cool, just making sure this is deliberate change

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**